tools/StaticAnalysis/IssueChecker/IssueChecker.cs

```diff
@@ -114,6 +115,10 @@ private bool IsSingleExceptionFileHasCriticalIssue(string exceptionFilePath, str
                         hasError = true;
                         errorText.AppendLine(record.FormatRecord());
                     }
+                    else if (record.Severity > 1 && outputWarning)
```

Comment:
I think we should not show informational issues but only show warnings, because the purpose of displaying these issues is to ask the PR author to fix them, but informational issues are not meant to be fixed.
```suggestion
                    else if (record.Severity == 2 && outputWarning)
```

Reply:
Agree